Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-main.view.html

```diff
@@ -88,7 +88,7 @@ <h4 id="systemUsageHeaderName">{{systemUsageName}}</h4>
                            ng-blur="patchDate('concluded', usage.concluded)"
                            data-field="concluded">
                 </div>
-                <div class="form-group">
+                <div class="form-group" ng-if="showDateFields">
```

Comment:
not dataFields --> usagePeriod... datefields is also a bad word when used in danish bit it is what they want... we should however keep our wording closer to the actual meaning of the data